app/domain/service-request/functions.server.ts

```diff
@@ -30,7 +31,7 @@ export async function getIndexedServiceRequest(address: EvmAddress, id: string):
     await upsertIndexedServiceRequest(address, id);
     return getIndexedServiceRequest(address, id);
   }
-  return doc;
+  return ServiceRequestWithIndexDataSchema.parse(doc);
```

Comment:
This line actually breaks the expiration dates because it tries to transform the dates as unix timestamps and you get bad data

Reply:
ohh shoot ok interesting. Thought I missed it before and was being safe. Let me delete

Reply:
At this point it _should_ be good data that we get back from mongo